bitnami/clickhouse-operator/values.yaml

```diff
@@ -0,0 +1,844 @@
+      ##
+      adaptSecurityContext: auto
+    ## @param global.compatibility.omitEmptySeLinuxOptions If set to true, removes the seLinuxOptions from the securityContexts when it is set to an empty object
+    ##
+    omitEmptySeLinuxOptions: false
+
+## @section Common parameters
+##
+
+## @param kubeVersion Override Kubernetes version
+##
+kubeVersion: ""
+## @param apiVersions Override Kubernetes API versions reported by .Capabilities
+##
+apiVersions: []
+## @param nameOverride String to partially override common.names.name
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride: ""
+## @param namespaceOverride String to fully override common.names.namespace
+##
+namespaceOverride: ""
+## @param commonLabels Labels to add to all deployed objects
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+
+## @section ClickHouse Operator Parameters
+
+## Bitnami ClickHouse Operator image
+## ref: https://hub.docker.com/r/bitnami/clickhouse-operator/tags/
+## @param image.registry [default: REGISTRY_NAME] ClickHouse Operator image registry
+## @param image.repository [default: REPOSITORY_NAME/clickhouse-operator] ClickHouse Operator image repository
+## @skip image.tag ClickHouse Operator image tag (immutable tags are recommended)
+## @param image.digest ClickHouse Operator image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag image tag (immutable tags are recommended)
+## @param image.pullPolicy ClickHouse Operator image pull policy
+## @param image.pullSecrets ClickHouse Operator image pull secrets
+## @param image.debug Enable ClickHouse Operator image debug mode
+##
+image:
+  registry: docker.io
+  repository: bitnami/clickhouse-operator
+  tag: 0.24.5-debian-12-r0
+  digest: ""
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: https://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  debug: false
```

Comment:
This is not being used. Maybe using it for setting a different log level?

Reply:
I'll remove the parameter

Reply:
done at https://github.com/bitnami/charts/pull/32816/commits/f07d8c5a0c14c4e10078b83f623887f6736a00ff